client/components/Product.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react'
+
+export default class Product extends Component {
+  render() {
+    const {product} = this.props
+
+    return (
+      <div>
+        <h3>{product.title}</h3>
+        <p>Price: ${product.priceCents / 100}</p>
```

Comment:
Consider making this formula a Redux selector function (which does derived calculations) – and/or, create a Sequelize virtual field (a field on the model which doesn't exist in the database, it is calculated from the db info).